src/pages/Autolending/CheckList.vue

```diff
@@ -6,9 +6,9 @@
 		>
 			<kv-toggle
 				class="list-item"
-				:name="id"
+				:id="`${id}`"
```

Comment:
can't we just use "id" here seeing as it's bound already... or is this to ensure it's cast to a string?

Reply:
Yeah, this is just casting it to a string.

Reply:
We should change this in the Checklist component. Right now it renders: id="<number>" and similarly in the for attribute, I think these should be descriptive, for the sake of automation testing as well. It should render an identifiable string, based on the name. So something like id="<name>" The name is unfortunately not slug friendly. See: src/components/WwwFrame/LendMenu/LendListMenu.vue Line 31